impl/src/main/java/org/ehcache/impl/internal/spi/copy/DefaultCopyProvider.java

```diff
@@ -54,6 +54,13 @@ public DefaultCopyProvider(DefaultCopyProviderConfiguration configuration) {
     return createCopier(DefaultCopierConfiguration.Type.VALUE, clazz, serializer, configs);
   }
 
+  @Override
+  public void releaseCopier(Copier<?> copier) throws Exception {
+    if (!(copier instanceof IdentityCopier) & !(copier instanceof SerializingCopier)) {
```

Comment:
Because these are the only ones that gets provided as default and its does not get instantiated by the instance provider